### PR TITLE
Pr 4 of 4: Add inline comments where users need to replace code

### LINKED ISSUE #99: Add inline comments in notebooks where users need to replace code
Users didn't always feel confident about what they needed to replace. Users tended to skim through the comments before the code blocks and often missed what they need to do. 

Add inline comments like
`Replace path with path to your data file`
`Replace with your organism` 
So it explicit what they are expected to do. 

Screen Shot 2020-05-29 at 9 19 26 AM

### PR DESCRIPTION
**Pull request by cbethell**
## Purpose:
What issue(s) does your PR address?

This PR addresses issue #99  by adding inline comments in the notebooks in this repository where users would need to replace code to tailor the notebooks for use with different data files.

This PR adds inline comments to the notebooks in the `validate-differential-expression` and `normalize-own-data` directories. 
Please let me know if I missed any sections that should have an inline comment added or if the wording I added is not clear.

**Note** that I was unable to rerun the `normalize-own-data/quantile_normalize_own_data.Rmd` notebook due to an error encountered at the line 

`qn_target_info <- jsonlite::fromJSON("https://api.refine.bio/qn_targets/?organism=DANIO_RERIO”)`

The error encountered is as follows:
`Error in open.connection(con, "rb") : HTTP error 500.`
(which appears to be an error with the connection to the refinebio server?)

## Results
If applicable, briefly report your findings.

This PR includes updates to the results files in the validate-differential-expression directory, which again I believe reflect updates made to the input files (so this should be fine but _again_ please correct me if I am wrong). Should the seed be set in the `validate-differential-expression` directory?

This closes #99 as PR #120 removes the last directory that was left to be updated with inline comments, `compare-processing`.

## Pull Request Check List:
* [x] Run a linter
* [ ] Set the seed (if applicable)
* [x] Comments and/or documentation up to date
* [x] Double check your paths
* [x] Check headline numbering
* [x] Spell check any Rmd file or md file
* [x] Restart R and run all notebooks fresh and save

### CODE REVIEW

**Comment by jaclyn-taroni** [comment]
Came up in meeting - related: #96

**Comment by cbethell** [comment]
> Note that I was unable to rerun the normalize-own-data/quantile_normalize_own_data.Rmd notebook due to an error encountered at the line

> qn_target_info <- jsonlite::fromJSON("https://api.refine.bio/qn_targets/?organism=DANIO_RERIO”)

> The error encountered is as follows:
> Error in open.connection(con, "rb") : HTTP error 500.
> (which appears to be an error with the connection to the refinebio server?)

The above mentioned issue was fixed using the information in @jaclyn-taroni's [comment above](https://github.com/AlexsLemonade/refinebio-examples/pull/121#issuecomment-656274100).
I was able to run `normalize-own-data/quantile_normalize_own_data.Rmd ` with the changes made in the last commit.

File: validate-differential-expression/gene_DE_validate.Rmd [hunk 1]
```diff
@@ -114,8 +124,9 @@ Note that these differential expression results come with Affymetrix probe IDs
 
 ```{r}
 # Load in the summary statistics from limma run
-author.full.results <- readr::read_tsv(file.path("results", 
-                                                 "GSE37382_limma_results.tsv")) 
+author.full.results <- readr::read_tsv(file.path(results_dir, 
+                                                 "GSE37382_limma_results.tsv" # Replace with name of `limma` results file generated in `author_processed_DE.Rmd`
```
Comment (cansavvy) [inline validate-differential-expression/gene_DE_validate.Rmd:128]:
```suggestion
                                                 "GSE37382_limma_results.tsv" # Replace with name of your own differential expression results file you've generated that you'd like to compare to refinebio data
```
We should make it more clear that it doesn't necessarily need to be this file. The idea of this example is that people can provide their own limma results file to compare with refinebio results. (the `author_processed_DE.Rmd` notebook is just a set up notebook that we used as an example of a file you could compare to, but ideally the user would have some sort of file like this already themselves). 

This being said, we should add more comments at places where metadata or column names for `author.full.results` might differ when our user uploads their own file. Just so they have more heads up where things are likely not to work for every file. (We can't guarantee their limma results file will look the same as ours).

File: validate-differential-expression/gene_DE_validate.Rmd [hunk 2]
```diff
@@ -176,7 +189,7 @@ variable `direction` we made in the previous section.
 ```{r}
 # Summarize the direction of significant probes for each gene
 direction.summary <- author.full.results %>% 
-  # Keep only significant at 0.05 probes
+  # Keep only significant at 0.05 probes -- you may need to replace <adj.P.Val>      # with the name of the column that contains the adjusted P values for your data
```
Comment (cansavvy) [inline validate-differential-expression/gene_DE_validate.Rmd:192]:
```suggestion
  # Keep only significant at 0.05 probes -- you may need to replace "adj.P.Val" with the corresponding column name in your data that contains the adjusted P values. 
```

File: validate-differential-expression/gene_DE_validate.Rmd [hunk 3]
```diff
@@ -145,10 +156,12 @@ probe.conversion <- readr::read_tsv(id.conversion.file,
 
 # Add annotation to imported author limma results 
 author.full.results <- author.full.results %>%
+  # You may need to replace "affy_probe_ids" with the name of the column that        # contains the probe IDs in your data 
   dplyr::inner_join(probe.conversion, by = c("affy_probe_ids" = "PROBEID")) %>%
   dplyr::mutate("ENSEMBL" = as.factor(ENSEMBL)) %>% 
   # Determine direction of fold change. 
   # Note up-directed genes will be TRUE, down-genes will be FALSE
+  # You may need to replace <logFC> with the name of the column that contains the    # log fold change values in your data
```
Comment (cansavvy) [inline validate-differential-expression/gene_DE_validate.Rmd:164]:
```suggestion
  # You may need to replace "logFC" with the name of the column that contains the    
  # log fold change values in your data 
```

File: validate-differential-expression/gene_DE_validate.Rmd [hunk 4]
```diff
@@ -145,10 +156,12 @@ probe.conversion <- readr::read_tsv(id.conversion.file,
 
 # Add annotation to imported author limma results 
 author.full.results <- author.full.results %>%
+  # You may need to replace "affy_probe_ids" with the name of the column that        # contains the probe IDs in your data
```
Comment (cansavvy) [inline validate-differential-expression/gene_DE_validate.Rmd:159]:
```suggestion
  # You may need to replace "affy_probe_ids" with the name of the column that        
  # contains the probe IDs in your data 
```

**Pull request merged**